pkg/storage/db.go

```diff
@@ -81,57 +81,56 @@ const latestStatusQuery = "WITH latest_candidates AS ( " +
 
 // Query to find resource by private IP using v2 schema
 // nolint
-const resourceByPrivateIPQuery = `select ia.private_ip,
-       res.arn_id,
-       res.meta,
-       ar.region,
-       rt.resource_type,
-       aa.account
-from aws_private_ip_assignment ia
-         left join aws_resource res on ia.aws_resource_id = res.id
-         left join aws_region ar on res.aws_account_id = ar.id
-         left join aws_resource_type rt on res.aws_resource_type_id = rt.id
-         left join aws_account aa on res.aws_account_id = aa.id
-where ia.private_ip = $1
-  and ia.not_before < $2
-  and (ia.not_after is null or ia.not_after > $2)
-`
+const resourceByPrivateIPQuery = "SELECT ia.private_ip, " +
```

Comment:
For clarification, all sql queries are declared as string because constant cannot be passed as sql query.

Reply:
question: weren't these already string literals?(the backtick ` indicates string literal) 

Why do they only need to be converted now if they were working as is? 

My source for thinking this doesn't need to be changed: https://stackoverflow.com/a/39025049

Reply:
personally I prefer backticks over needing to add `" +` to most lines of the string

Reply:
I figured out that I had misunderstood the error which I encountered. In test, the ExpectQuery(query string) throws an error because of case sensitivity, so I have modified argument(change "SELECT" to "select"). Thank you for your explanation.